dids/src/main/kotlin/web5/sdk/dids/extensions/DidExtensions.kt

```diff
@@ -0,0 +1,27 @@
+package web5.sdk.dids.extensions
+
+import foundation.identity.did.DID
+import web5.sdk.crypto.KeyManager
+import web5.sdk.dids.Did
+import web5.sdk.dids.methods.dht.DidDht
+import web5.sdk.dids.methods.ion.DidIon
+import web5.sdk.dids.methods.jwk.DidJwk
+import web5.sdk.dids.methods.key.DidKey
+import web5.sdk.dids.methods.web.DidWeb
+
+internal val supportedMethods = mapOf(
+  DidKey.methodName to DidKey.Companion,
+  DidJwk.methodName to DidJwk.Companion,
+  DidIon.methodName to DidIon.Default,
+  DidDht.methodName to DidDht.Default,
+  DidWeb.methodName to DidWeb.Default
+)
```

Comment:
This looks like the same values as `methodResolves` in the `DidResolvers.kt` class. Wondering if it might make sense to combine them?

Reply:
Combined!